### PR TITLE
Added template for pull requests #109196

### PR DESCRIPTION
**Pull request by Alex-302**
Created template for pull requests https://github.com/AdguardTeam/AdguardFilters/issues/109196

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+### Prerequisites
+##### To avoid invalid pull requests, please check and confirm following checkboxes:
+
+        
+  - [ ] This is not an ad/bug report;
+  - [ ] My code follows the style guidelines and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
+  - [ ] I have performed a self-review of my own changes;
+  - [ ] My changes do not break web sites, apps and files structure.
+
+### What what problem does pull request fix?
```
Comment (zloyden) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:18]: > What what

Maybe typo?

Comment (Alex-302) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: fixed

**Review by zloyden — COMMENTED** [review]
I'd also add that if an account is required to reproduce the issue, it should be provided to our email.

**Comment by Alex-302** [comment]
But this is for user's proposal to solve the problem.
If necessary, we may request additional information separately. Not sure if more questions is a good idea(too much to read).

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+---
+
+
+### Prerequisites
+##### To avoid invalid pull requests, please check and confirm following checkboxes:
+
+        
+  - [ ] This is not an ad/bug report;
+  - [ ] My code follows the style guidelines and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
```
Comment (ameshkov) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]:
```suggestion
  - [ ] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  - [ ] I have performed a self-review of my own changes;
+  - [ ] My changes do not break web sites, apps and files structure.
+
+### What problem does pull request fix?
+##### If the problem does not fall under any category that is listed here, please contact our tech support - support@adguard.com
```
Comment (ameshkov) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:19]:
```suggestion
##### If the problem does not fall under any category that is listed here, please contact our tech support: `support@adguard.com`
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+  - [ ] Missed ads or ad leftovers;
+  - [ ] Website or app doesn't work properly;
+  - [ ] AdGuard gets detected on a website;
+  - [ ] Missed analytics or tracker;
+  - [ ] Social media buttons — share, like, tweet, etc;
+  - [ ] Annoyances — pop-ups, cookie warnings, etc.
+  
+### Which issue is fixed?
```
Comment (ameshkov) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:28]:
```suggestion
### What issue is being fixed?
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+##### Enter an issue address:
```
Comment (ameshkov) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:29]:
```suggestion
##### Enter the issue address:
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+### Add your comment and screenshots
+##### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located.
+
+**0.** **DO NOT** upload screenshots with sexually explicit material on GitHub directly. Instead, upload it to third-party image hosting and post URL without preview here;
+
+**1.** Add screenshots of the problem. You can drag and drop images or paste them from clipboard;
+
+Use `<details> </details>` tag to hide screenshots under the spoiler; 
+
+**2.** Describe the issue extensively unless it is absolutely clear from the screenshot what the problem is;
```
Comment (ameshkov) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:42]:
```suggestion
**2.** Describe the issue in detail until it is absolutely clear from the screenshot what the problem is;
```

**Review by zloyden — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**